packages/mux-player/src/media-theme-mux.html

```diff
@@ -530,7 +531,7 @@
           </media-control-bar>
         </template>
 
-        <template if="targetLiveWindow == null">
+        <template if="targetLiveWindow == 0">
```

Comment:
note: This change is based on the m-ui-ext spec (See: https://github.com/video-dev/media-ui-extensions/issues/4#issuecomment-1344924246). We may want to discuss theme compatibility complexities as a result. i.e.: This is no longer compatible with slotted media elements that don't implement `targetLiveWindow`. Fine for mux-video/mux-player but has implications for our new themes that "live" in media-chrome (e.g. micro theme)

Reply:
Solved in media chrome per recommendation 🎉